README.md

```diff
@@ -40,101 +37,25 @@ devtools::install_github("MoTrPAC/MotrpacBicQC", build_vignettes = TRUE)
 
 # Usage
 
-First, load the library
-
-```
-library(MotrpacBicQC)
-```
-
-And run any of the following tests to check that the package 
-is correctly installed it works. For example:
-
-```
-# Just copy and paste in the RStudio terminal
-
-check_metadata_metabolites(df = metadata_metabolites_named, name_id = "named")
-check_metadata_samples(df = metadata_sample_named, cas = "umichigan")
-check_results(r_m = results_named, m_s = metadata_sample_named, m_m = metadata_metabolites_named)
-```
+[Follow this link](https://motrpac.github.io/MotrpacBicQC/articles/metabolomics_qc.html) to learn how to test and use this package.
 
-To find out more details, open the package's vignette:
+Alternatively, once the package is installed, run the following command to 
+access the same documentation:
 
 ```
 browseVignettes("MotrpacBicQC")
 ```
 
-## How to test your datasets
-
-Two approaches available:
-
-### Check full `PROCESSED_YYYYMMDD` folder
-
-Run test on the full submission. For that, run the following command:
-
-```
-validate_metabolomics(input_results_folder = "/full/path/to/PROCESSED_YYYYMMDD", 
-                      cas = "your_site_code")
-```
-
-**cas** is one of the followings:
-
-- "broad_met" = Broad Metabolomics
-- "emory"     = Emory
-- "mayo"      = Mayo Clinic
-- "umichigan" = Umichigan
-- "gtech"     = Georgia Tech
-- "duke"      = Duke
-
-### Check individual files
-
-- Check metadata metabolites:
-
-```
-# Open the metadata_metabolites file(s)
-
-metadata_metabolites_named <- read.delim(file = "/path/to/your/file", stringsAsFactors = FALSE)
-metadata_metabolites_unnamed <- read.delim(file = "/path/to/your/file", stringsAsFactors = FALSE)
-
-check_metadata_metabolites(df = metadata_metabolites_named, name_id = "named")
-check_metadata_metabolites(df = metadata_metabolites_unnamed, name_id = "unnamed")
-
-```
-
-- Check metadata samples:
-
-```
-# Open your files
-metadata_sample_named <- read.delim(file = "/path/to/your/file", stringsAsFactors = FALSE)
-metadata_sample_unnamed <- read.delim(file = "/path/to/your/file", stringsAsFactors = FALSE)
-
-check_metadata_samples(df = metadata_sample_named, cas = "your_side_id")
-check_metadata_samples(df = metadata_sample_unnamed, cas = "your_side_id")
-```
-
-- Check results, which needs both both metadata metabolites and samples
-
-```
-# Open your files
-metadata_metabolites_named <- read.delim(file = "/path/to/your/file", stringsAsFactors = FALSE)
-metadata_sample_named <- read.delim(file = "/path/to/your/file", stringsAsFactors = FALSE)
-results_named <- read.delim(file = "/path/to/your/file", stringsAsFactors = FALSE)
-
-check_results(r_m = results_named, 
-              m_s = metadata_sample_named, 
-              m_m = metadata_metabolites_named)
-```
+[Follow this link](https://motrpac.github.io/MotrpacBicQC/reference/index.html) 
```

Comment:
(Minor anal-retentive point, not worth fixing for PR but to make you aware) - the function descriptions in this link sometimes start with Uppercase and sometimes lowercase.